### DISCUSSION TITLE
Renault Zoe - Updated ECU?

### DISCUSSION THREAD

**Discussion by lilltonka** [cedricp/ddt4all#1172] [Q&A] [answered]
Hi! First of all, thanks for making such a great tool! I'm however experiencing issues when trying to enable the Rear Camera in my Renault Zoe. I have tried the looking glass function, but I'm not really understanding the issue. Se attached screen shot.

Am I doing something wrong? I have both an ELM327 Bluetooth and USB version.

![Skärmbild 2024-07-23 131504](https://github.com/user-attachments/assets/c6c28d3b-1e51-4fdd-9d47-869cbb641d2a)

**Comment by AndyIreland** [comment]
Hi, it wont find it using the magnifying glass you have to select the MFD manually:
![image](https://github.com/user-attachments/assets/715f324d-542f-4499-8550-65452a1f3991)

**Reply by lilltonka** [reply]
Thanks for the reply!

I have tried that, but I only get a lot of zero as response when trying to write the changes. And trying the hard reset at the top does nothing.

**Comment by AndyIreland** [comment]
what year is your zoe if its a later one it may be secured by security gateway on the OBD 2 port to block changes and need a token to unlock . 
How about your ELM327 is it compatible with DDT4ALL? with the correct PIC chip?
How have you wired the ELM327 to read the R-link through the OBD2 socket? - did you re solder ELM327 or are you using a specially wired adapter cable?

**Reply by lilltonka** [reply]
The car is from 2015, so it should not be blocked by the gateway. According to the site I bought the USB ELM327 plug from, it is using the correct chip. The Bluetooth one I can't really tell, but it is more than 5 years old. I have been trying to use both of them, both using the Zoe OBD2 port near the gearstick inside the car.

**Reply by AndyIreland** [reply]
no, 2015 wouldnt have security on the OBD2 socket. 
How did you wire up the ELM327 to read the rlink through the OBD2 socket? did you re solder the ELM327 or did you re-wire a OBD2 cable splitter?

**Reply by lilltonka** [reply]
Thanks for your efforts! 

The last part in your answer I'm a bit puzzled about. I have just put the OBD plug in the port, nothing else. Do I need to alter the plug itself or have a specific cable? It is using USB to the laptop.

**Reply by AndyIreland** [reply, marked as answer]
yes , you do have to make the modification to it. 
Read in this article it tells you what pins to re-solder so you can read/write parameters to the r-link through OBD2 socket:
https://www.gps-rlink.com/r-link-1/activate-android-auto-waze-on-r-link-1/

**Reply by lilltonka** [reply]
Thank you @AndyIreland! I totally missed that part! Will take a look at it to make the changes!

**Comment by vicentmartinezmartinez** [comment]
yo tengo el mismo problema.
en el buscador puedo encontrar diferentes ecus, pero cuando intento mfd v5.4  no me deja modificar los datos.
He probado con diferentes obd2 aunque solo este ha podido encontrar diferentes ecs en el buscador.
Este obd2 si permite modificar las ecus que encuentra, pero no las mfd v5.4 , los datos estan en rojo.

Lo siguiente que me queda es modificar el cable.

https://www.amazon.es/gp/product/B07MQ8GHG3/ref=ppx_yo_dt_b_asin_title_o00_s01?asc_source=01H8HFYCRM99TJ9FED7FSB1ZXC&ie=UTF8&psc=1&tag=snxes62-21

**Reply by AndyIreland** [reply]
This is the cable you would be looking at getting and modifying and using with your ELM:
https://www.amazon.es/bbfly-B6-divisor-extensi%C3%B3n-Cable-adaptador/dp/B073Y519FG/ref=sr_1_3?dib=eyJ2IjoiMSJ9.J1Z_HrVfVmkNfxtCuwk9ZizP4yPUIKcDZoEGrepz7QDt9tgD9WWONQH9Hskpor103NP4TO-nZ4AssyLvGOcm08LpVJhfzeB67XHfvfdPC5KpXIe1ZmST5NPxrKlVn33J.l0Ldt1AwXbIjvGxVwlV28XFY1n9zCbi7MxMQw9IgG9U&dib_tag=se&keywords=bbfly%2Bcable&qid=1722795390&sr=8-3&th=1

![image](https://github.com/user-attachments/assets/8e6ad3c9-cb12-4e22-86f8-15ed28d2dd46)

and wire it up like in this:
https://www.gps-rlink.com/r-link-1/activate-android-auto-waze-on-r-link-1/
![image](https://github.com/user-attachments/assets/f371b45b-8aec-4377-bcfc-c7720fd3b3a8)

**Comment by vicentmartinezmartinez** [comment]
Thanks very much
    En domingo, 4 de agosto de 2024, 20:23:12 CEST, Andy Robus ***@***.***> escribió:  
 
 


This is the cable you would be looking at getting and modifying and using with your ELM:
https://www.amazon.es/bbfly-B6-divisor-extensi%C3%B3n-Cable-adaptador/dp/B073Y519FG/ref=sr_1_3?dib=eyJ2IjoiMSJ9.J1Z_HrVfVmkNfxtCuwk9ZizP4yPUIKcDZoEGrepz7QDt9tgD9WWONQH9Hskpor103NP4TO-nZ4AssyLvGOcm08LpVJhfzeB67XHfvfdPC5KpXIe1ZmST5NPxrKlVn33J.l0Ldt1AwXbIjvGxVwlV28XFY1n9zCbi7MxMQw9IgG9U&dib_tag=se&keywords=bbfly%2Bcable&qid=1722795390&sr=8-3&th=1

image.png (view on web)

and wire it up like in this:
https://www.gps-rlink.com/r-link-1/activate-android-auto-waze-on-r-link-1/
image.png (view on web)

**Reply by vicentmartinezmartinez** [reply]
muchas gracias, hoy lo pruebo
